test_settings.py

```diff
@@ -61,3 +61,5 @@ def root(*args):
 }]
 
 FORUM_PORT = "4567"
+MONGO_HOST = "localhost"
```

Comment:
Could we set a bogus URL here to make sure that we don't accidentally hit the real mongodb in unit tests?

Reply:
done